database/factories/ModelFactory.php

```diff
@@ -479,7 +467,7 @@
 });
 
 // Child - over School Age
-$factory->defineAs(App\Child::class, 'overSchoolAge', function (Faker\Generator $faker) {
+$factory->defineAs(App\Child::class, 'isPrimarySchoolAge', function (Faker\Generator $faker) {
```

Comment:
I will have to change the test seeds I've created to reflect these changes or we can do so in this PR maybe...